utils/hostdoc/hostdoc_test.go

```diff
@@ -37,37 +37,37 @@ func TestProcess(t *testing.T) {
 	}{
 		{
```

Comment:
why do you have changes in this file?

Reply:
I think I have some auto-formatting that caused it